### PR TITLE
Fix layout

### PR DESCRIPTION
**Pull request by Shipow**
Some tweaks regarding html5 markup

> The content inside the _main_ element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms.

Semantic for containers: 
Left column -> sidebar
Right column -> results

Some css stuff:
Ordered rules by impact on the layout: from box to style (ie: float is more important than color)

### CODE REVIEW

**Comment by redox** [comment]
:+1:

File: app/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
```
Comment (vvo) [inline app/index.html:1]: Only for clarification: this is not a requirement of any HTML validator right? both are equivalent (and lowercase is always the easiest: no question about case)

Comment (Shipow) [inline app/index.html:1]: yes collateral damage, I was hunting something else ;)

**Comment by vvo** [comment]
giphy 2

File: app/index.html (hunk 1 above)
Comment (vvo) [inline app/index.html:1]: giphy 2

**Pull request merged**